packages/@aws-cdk/aws-sqs/lib/perms.ts

```diff
@@ -0,0 +1,7 @@
+export const QUEUE_GET_ACTIONS = [
+    "sqs:ReceiveMessage",
```

Comment:
Should have `DeleteMessage(s)` and `SetVisibilityTimeout` (and maybe others?) as well I think. Those are common receiver actions.

Reply:
I based this off the templated permissions the user expects in the console and the examples that the IAM documentation states are needed (doesn't mean it's right in this case!). Maybe more detailed grants might work for those cases?